doc/02-Installation.md

```diff
@@ -26,14 +37,24 @@ GRANT SELECT, INSERT, UPDATE, DELETE, DROP, CREATE VIEW, INDEX, EXECUTE ON x509.
 ```
 
 After, you can import the schema using the following command:
+<!-- {% if not icingaDocs %} -->
+
+**Note:** If you haven't installed this module from packages, then please adapt the schema path to the correct installation path.
+
+<!-- {% endif %} -->
```

Comment:
I hope users are not that oblivious that we need to repeat this 10 lines later again.